service/service.go

```diff
@@ -309,7 +310,7 @@ func (s *Service) GetUser(ctx context.Context, req *pb.UserRequest) (*pb.UserRes
 		return nil, consts.StatusUUIDNotFound
 	}
 
-	logger.Info("Retrieved user:", user.GetUuid())
+	logger.Info("Retrieved user:", user.GetUuid(), user.GetFirstName(), user.GetLastName())
 
 	return &pb.UserResponse{
```

Comment:
`updateUserRow` <- I am really against mutating. 
- it makes it hard to understand the code
- debug can be confusing
- maintaining with other people